### DISCUSSION TITLE
ASUS A17 FA706IC - what works and doesn't appear to work.

### DISCUSSION THREAD

**Discussion by steverand** [seerge/g-helper#342] [General]
Again, thanks for the excellent tool. I've been running with no AC for about a year and I have pretty much every ASUS service disabled. Prior to GHelper I had the keyboard set to dimmest/white/always on, dgpu to always on, screen refresh auto 60/144.

v0.58
G-Helper can read my CPU temp and fan but cannot change the fan curve at all. (not supported by laptop)
Performance modes work although I leave it on balanced
GPU mode works, GPU temp and fan readout works. GPU fan curve does not work at all. (not supported by laptop)
Laptop screen switch works on AC/battery
Laptop keyboard backlight mode and colour work, key bindings do nothing and are not correct for my model. Awake/boot/sleep/shutdown appears to work. Brightness control works except if I choose to lower backlight on battery and i'm on the dimmest setting it will lower it to "off" entirely.
Screen overdrive - not sure, untested and not sure if my laptop does more than 144
GPU disabled on usb-c - untested. not sure if my laptop takes a charge from usb-c
Battery limit - does not work. It appears that setting the slider anywhere will update the registry key to 100% and I have to manually set it back to 80. The slider does not reflect the registry key either.

On my laptop:
Fn-F5 swaps performance modes
Fn-up/down raises and lowers keyboard backlight
Fn-left/right changes Aura modes but only right appears to work without AC installed. Fn-left does nothing.
![2023-05-09 15-23-46 G-Helper](https://user-images.githubusercontent.com/113805096/237002177-610a54c9-1782-4870-96db-a988f35c68f0.png)
![2023-05-09 15-24-09 G-Helper](https://user-images.githubusercontent.com/113805096/237002199-3b68a373-5b4e-4864-871c-bedd7a8a66c3.png)
![PXL_20230509_052457830](https://user-images.githubusercontent.com/113805096/237002280-dbe0c79b-6d32-4961-a00b-2cf424decf6c.jpg)
![PXL_20230509_052506118](https://user-images.githubusercontent.com/113805096/237002288-abec7336-b2b1-4279-892a-f79bd4595d24.jpg)
![PXL_20230509_052521272](https://user-images.githubusercontent.com/113805096/237002293-d4d25663-c8b2-4c64-9653-2bb35aace46f.jpg)

**Comment by steverand** [comment]
With later releases, keyboard backlight timeout works. I've got it set to 0 to never time out.
Battery charge limit still not working for me, but it does if I change the registry manually.
I'll look at the services again, the only ASUS service I have enabled is ASUS Optimization, which is for the keyboard shortcuts. (performance modes, aura, backlight etc)

**Comment by seerge** [comment]
@steverand what do you change in registry for battery charging ?

**Comment by steverand** [comment]
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\ASUS\ASUS System Control Interface\AsusOptimization\ASUS Keyboard Hotkeys
ChargingRate
Using decimal, set to percentage required.

**Reply by seerge** [reply]
Looks like one more (?) setting that asus optimization service does in the system next to this https://github.com/seerge/g-helper/discussions/361 , probably for old laptops

**Reply by steverand** [reply]
Yeah I was just looking at that thread. I'll see if I have that configuration file too. Something is resetting the value as you had suggested to me previously. Thanks for looking into it!

**Reply by seerge** [reply]
@steverand new version https://github.com/seerge/g-helper/releases/tag/v0.65 can handle all main hotkeys, so you may completely stop asus optimization service (so it won't overwrite any charge settings)
